src/hnsw/htab_cache.h

```diff
@@ -0,0 +1,27 @@
+#ifndef LDB_HNSW_HTAB_CACHE_H
+#define LDB_HNSW_HTAB_CACHE_H
+#include "postgres.h"
+
+#include <utils/hsearch.h>
+#include <utils/memutils.h>
+
+typedef int HTABCacheKey;
```

Comment:
You should use sized integers always (e.g. int32)
Would be great if we could add a ci/cd check for this kinds of things

Reply:
Changed to int32 for now, will take a not for CI checks